openstack/nova/templates/hypervisors/_ironic-deployment.yaml.tpl

```diff
@@ -24,17 +24,18 @@ spec:
   template:
     metadata:
       labels:
-{{ tuple . "nova" "compute" | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
+{{ tuple . "nova" $compute_hypervisor_name | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
         name: nova-compute-{{$hypervisor.name}}
         hypervisor: "ironic"
       annotations:
+        pod.beta.kubernetes.io/hostname: nova-compute-{{$hypervisor.name}}
```

Comment:
Please remove this

Reply:
This has been introduced by @galkindmitrii, please resolve this

Reply:
It looks to me more like your rebasing re-introduced an old change, which was superseeded by a later change from me

Reply:
I removed dmitrys commits, is that ok @galkindmitrii ?